azuracast/liquidsoap_custom.liq

```diff
@@ -409,76 +409,76 @@ radio = switch(track_sensitive=false, transitions=[
+    ({2w and 11h-12h29m}, once(playlist_csinaltam_neked_egy_valogatas_kazettat)),
+    ({2w and 14h-14h59m}, once(playlist_mills_weather)),
+    ({2w and 15h-15h59m}, once(playlist_azvlm)),
+    ({2w and 16h-17h29m}, once(playlist_mmn_radio)),
+    ({2w and 17h30m-18h29m}, once(playlist_havizaj)),
+    ({2w and 19h-19h59m}, once(playlist_turmeric_acid)),
+    ({2w and 20h-21h59m}, once(playlist_mosolyszunet)),
+    ({2w and 14h-20h30m}, off_air_ambient_mix),                        #ambient: Mill - Mosoly
+    ({3w and 10h-11h59m}, once(playlist_lahmacun_presents)),           #Wednesday
+    ({3w and 12h30m-12h59m}, once(playlist_tudtad)),
+    ({3w and 13h-13h59m}, once(playlist_erto_hallgatas)),
+    ({3w and 14h-14h59m}, once(playlist_hirszalonna)),
+    ({3w and 15h-15h59m}, once(playlist_gold_liebe)),
+    ({3w and 16h-16h59m}, once(playlist_svadhyaya)),
+    ({3w and 17h-17h59m}, once(playlist_rambo)),
+    ({3w and 18h-18h59m}, once(playlist_cellz)),
+    ({3w and 19h-19h59m}, once(playlist_eastern_daze)),
+    ({3w and 20h-20h59m}, once(playlist_1800venus)),
+    ({3w and 21h-21h59m}, once(playlist_radio_bluszcze)),
+    ({3w and 12h30m-21h30m}, off_air_ambient_mix),                        #ambient: Tudtad - Bluszcze
+    ({4w and 12h-12h59m}, once(playlist_muhelycast)),                  #Thursday
+    ({4w and 13h-13h59m}, once(playlist_alternoon)),
+    ({4w and 14h-14h59m}, once(playlist_himihumi)),
+    ({4w and 15h-16h59m}, once(playlist_lazy_calm_raga)),
+    ({4w and 17h-17h59m}, once(playlist_dalmata_gergo_show)),
+    ({4w and 18h-18h59m}, once(playlist_dodo)),
+    ({4w and 19h-20h59m}, once(playlist_great_galactick_ghaul)),
+    ({4w and 21h-22h29m}, once(playlist_gyogyfurdo)),
+    ({4w and 14h-21h30m}, off_air_ambient_mix),                        #ambient: Alter - Gyogyf
+    ({5w and 12h30m-12h59m}, once(playlist_melania)),                     #Friday
+    ({5w and 13h-13h59m}, once(playlist_boombap)),                     
+    ({5w and 14h-14h59m}, once(playlist_recently_played)),             
+    ({5w and 15h-15h59m}, once(playlist_dead_hound)),
+    ({5w and 16h-16h59m}, once(playlist_agybaj)),
+    ({5w and 17h-17h59m}, once(playlist_mukodo_mukod)),
+    ({5w and 18h-18h59m}, once(playlist_moneyka)),
+    ({5w and 19h-20h59m}, once(playlist_arcadeok_alatt)),       
+    ({5w and 14h-19h30m}, off_air_ambient_mix),                        #ambient: Melánia - Arcade
+    ({6w and 09h30m-11h29m}, once(playlist_merites)),                  #Saturday
+    ({6w and 11h30m-12h29m}, once(playlist_jambikus_nesz)),     
+    ({6w and 12h30m-14h59m}, once(playlist_torso)),
+    ({6w and 15h-15h59m}, once(playlist_footwork_jimbob)),
+    ({6w and 16h-17h59m}, once(playlist_rnr666)),
+    ({6w and 18h-19h59m}, once(playlist_mood_sequence)),
+    ({6w and 20h00m-20h59m}, once(playlist_brains_sublimating)),
+    ({6w and 21h00m-22h29m}, once(playlist_d23)),
+    ({6w22h30m-6w23h29m}, once(playlist_linear_systems_with_gestalt)),
```

Comment:
We could have the other syntax now, to be consistent, but I agree, it's better to keep it like that so that we don't have to remember how it was when (if) we need to change it back again.